Spanner/src/Connection/Grpc.php

```diff
@@ -120,11 +122,25 @@ class Grpc implements ConnectionInterface
      */
     private $longRunningGrpcClients;
 
+    private function enableConnectionManagement($conf_path)
+    {
+        // TODO(ddyihai): move this function to GrpcTrait, if we are going
+        // to enable the grpc-gcp library for other apis.
+        $conf = new ApiConfig();
+        $conf->mergeFromJsonString($string = file_get_contents($conf_path));
```

Comment:
It doesn't look like we need to declare the variable `$string` - it isn't used anywhere.